# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/ash/src/vk/feature_extensions.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/ash/src/vk/feature_extensions.rs b/third_party/rust/ash/src/vk/feature_extensions.rs
--- a/third_party/rust/ash/src/vk/feature_extensions.rs
+++ b/third_party/rust/ash/src/vk/feature_extensions.rs
@@ -206,17 +206,17 @@ impl DependencyFlags {
 }
 #[doc = "Generated from 'VK_VERSION_1_1'"]
 impl StructureType {
     pub const PHYSICAL_DEVICE_VARIABLE_POINTERS_FEATURES: Self = Self(1_000_120_000);
 }
 #[doc = "Generated from 'VK_VERSION_1_1'"]
 impl StructureType {
     pub const PHYSICAL_DEVICE_VARIABLE_POINTER_FEATURES: Self =
-        StructureType::PHYSICAL_DEVICE_VARIABLE_POINTERS_FEATURES;
+        Self::PHYSICAL_DEVICE_VARIABLE_POINTERS_FEATURES;
 }
 #[doc = "Generated from 'VK_VERSION_1_1'"]
 impl StructureType {
     pub const PROTECTED_SUBMIT_INFO: Self = Self(1_000_145_000);
 }
 #[doc = "Generated from 'VK_VERSION_1_1'"]
 impl StructureType {
     pub const PHYSICAL_DEVICE_PROTECTED_MEMORY_FEATURES: Self = Self(1_000_145_001);
@@ -542,17 +542,17 @@ impl StructureType {
 }
 #[doc = "Generated from 'VK_VERSION_1_1'"]
 impl StructureType {
     pub const PHYSICAL_DEVICE_SHADER_DRAW_PARAMETERS_FEATURES: Self = Self(1_000_063_000);
 }
 #[doc = "Generated from 'VK_VERSION_1_1'"]
 impl StructureType {
     pub const PHYSICAL_DEVICE_SHADER_DRAW_PARAMETER_FEATURES: Self =
-        StructureType::PHYSICAL_DEVICE_SHADER_DRAW_PARAMETERS_FEATURES;
+        Self::PHYSICAL_DEVICE_SHADER_DRAW_PARAMETERS_FEATURES;
 }
 #[doc = "Generated from 'VK_VERSION_1_2'"]
 impl StructureType {
     pub const PHYSICAL_DEVICE_VULKAN_1_1_FEATURES: Self = Self(49);
 }
 #[doc = "Generated from 'VK_VERSION_1_2'"]
 impl StructureType {
     pub const PHYSICAL_DEVICE_VULKAN_1_1_PROPERTIES: Self = Self(50);